sdk/core/core-amqp/src/retry.ts

```diff
@@ -111,38 +111,36 @@ async function checkNetworkConnection(host: string): Promise<boolean> {
 }
 
 /**
- * It will attempt to linearly retry an operation specified number of times with a specified
- * delay in between each retry. The retries will only happen if the error is retryable.
+ * The retries will only happen if the an error occurred on the operation attempt and
+ * is classified as retryable.
  *
+ * The retries when made are done so linearly on the given operation for a specified number of times,
+ * with a specified delay in between each retry. 
```

Comment:
Am guessing that the reason to update was to make it clear about the +1 change we made? If so, it's still not clear :(

Reply:
This line just describes the linear retry mechanism.
The line before that should have described about how the retry function is implemented - but updated this some more in latest revision, hope that helps!